### PR TITLE
feat: common styles and layout

### PR DESCRIPTION
**Pull request by frimoldi**
- Added common styles like background and text color
- Created Footer component
- Created basic layout for header, content and footer.

### CODE REVIEW

File: src/App.module.scss [hunk 1]
```diff
@@ -1,3 +1,23 @@
+@import "styles/variables";
+
 .app {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  background-color: $background-color;
+  font-family: $font-family;
```
Comment (rcstanciu) [inline src/App.module.scss:9]: I suggest setting the default font family at the root level (index.css maybe), and override it when necessary.

File: src/styles/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+$font-family: "Inter";
```
Comment (rcstanciu) [inline src/styles/_variables.scss:1]: Will you load Inter font from Google Fonts CDN?

File: src/App.module.scss (hunk 1 above)
Comment (frimoldi) [reply src/App.module.scss:9]: yeah, good call.

File: src/styles/_variables.scss (hunk 2 above)
Comment (frimoldi) [reply src/styles/_variables.scss:1]: I thought about it when I wrote this exact line and then forgot about it, lol

**Review by rcstanciu — APPROVED** [review]

**Pull request merged**